recipes/busco/4.1.4/meta.yaml

```diff
@@ -60,6 +60,8 @@ about:
            near-universal single-copy orthologs selected from OrthoDB.
 
 extra:
```

Comment:
@skrakau can you please add a comment why this is needed. Ideally we remove this later, when the base container is fixed. Thanks.

Reply:
Hi @bgruening, 

the current default base image doesn't work with singularity, or more precisely requires to mount the hosts `/etc.resolv.conf` to `/tmp/resolv.conf` (instead of to `/etc.resolv.conf`) because of a dead link in busybox. In order to avoid that this needs to be specified for each pipeline or documented and known by each user of this container, I thought it would be good to adjust this recipe instead.

Reply:
(only for tools that need to download something)

Reply:
Oh sorry I meant into the recipe as comment. Sorry for the confusion.

Reply:
ah, yeah makes sense :)